From b533cc0d69f08c773038dd82c9e17c4286b090a8 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 17 Aug 2020 18:05:42 +0200
Subject: [PATCH 1566/1851] PCI: xdma: Use unique name for pci-xdma-pl.c

commit a3cb1eac84c545b3e7937f22744c16270161f8d3 from
https://github.com/Xilinx/linux-xlnx.git

xilinx-pcie name is used already by pcie-xilinx.c PL driver which is
causing error when both drivers are enabled.

Error in log:
[    3.716734] Error: Driver 'xilinx-pcie' is already registered, aborting...
[    3.717709] ps_pcie_dma init()

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/pcie-xdma-pl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/pcie-xdma-pl.c b/drivers/pci/controller/pcie-xdma-pl.c
index fd944cb323e1..102644fbf8eb 100644
--- a/drivers/pci/controller/pcie-xdma-pl.c
+++ b/drivers/pci/controller/pcie-xdma-pl.c
@@ -902,7 +902,7 @@ static const struct of_device_id xilinx_pcie_of_match[] = {
 
 static struct platform_driver xilinx_pcie_driver = {
 	.driver = {
-		.name = "xilinx-pcie",
+		.name = "xilinx-xdma-pcie",
 		.of_match_table = xilinx_pcie_of_match,
 		.suppress_bind_attrs = true,
 	},
-- 
2.31.1

